### PR TITLE
Cache invalidation

### PR DESCRIPTION
**Pull request by ASankaran**
- [x] Invalidate caches upon deployment

### CODE REVIEW

**Comment by YashoSharma** [comment]
I'd probably only start the cloudflare invalidation after the cloudfront is done (we never got around to ensuring this script was more consistent last year)

File: scripts/invalidate.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+if [ "$TRAVIS_BRANCH" = "master" ]
+then
+
+aws cloudfront create-invalidation --distribution-id $CLOUDFRONT_DISTRIBUTION --paths '/*' 2>&1
+
```
Comment (YashoSharma) [inline scripts/invalidate.sh]: These shouldn't run simultaneously

**Comment by ASankaran** [comment]
Will reopen once I get to properly setting this up.

**Comment by ASankaran** [comment]
Removed caching from cloudfront distribution, so the site is now only cache by cloudflare. On deploys to master the cloudflare cache will be invalidated.

**Pull request merged**